ports/zynq/Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: all run fclean clean build
+
+OPSET ?= $(shell bin/opset_list.sh)
+
+all: gen
+
+clean:
+	rm -rf gen
+
+fclean:
```

Comment:
I think this should be included on `clean`. Because `clean` command usually clean all generated files